kin-sdk-core/src/main/java/kin/sdk/core/KinAccount.java

```diff
@@ -69,4 +69,24 @@ TransactionId sendTransactionSync(String publicAddress, String passphrase, BigDe
      * @throws OperationFailedException any other error
      */
     Balance getBalanceSync() throws OperationFailedException;
+
+    /**
+     * Create {@link Request} for activating this account, should be done after account was created
+     * <p> See {@link KinAccount#activateSync(String)} for possibles errors</p>
+     *
+     * @param passphrase the passphrase used to create the account
+     * @return {@code Request<Void>}
+     */
+    Request<Void> activate(String passphrase);
+
+    /**
+     * Activate this account, should be done after account was created
```

Comment:
calling the client createAccount method is not enough for the account to be created
after the KinAccount object that contains KeyPair has been created we need to request from server to actually create the account and fund it with lumens. The documentation here is confusing